content/docs/replication/uninstall.md

```diff
@@ -32,7 +31,9 @@ If you used `controller.yaml` manifest with either `kubectl` or `repctl`, use th
 kubectl delete -f deploy/controller.yaml
 ```
 
-To delete the replication CRD, you can run the command:
+To delete the replication group CRD, you can run the command:
+
+All replication groups should be deleted before deleting the replication group CRD.
```

Comment:
The ordering of this is off. The command should appear right after 'you can run the command:', line 36 should be after the cmd itself.